solutions/cpp/01-oo8/code/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(shell-starter-cpp)
+
+file(GLOB_RECURSE SOURCE_FILES src/*.cpp)
```

Comment:
Can you add `src/*.hpp` too ?

Reply:
@ryan-gang Oh yup. Also as an aside, shouldn't we do `src/**/*.cpp` and`src/**/*.hpp` to also match any files in the subdirectories of `src/`? 

Currently it only matches files directly within `src/`.

Reply:
Worth confirming if this is the case, I read about this briefly yesterday and found a few resources saying just "*" would match subdirectories too 

<img width="510" alt="Screenshot 2024-06-06 at 10 53 27" src="https://github.com/codecrafters-io/build-your-own-shell/assets/3893573/24f124fa-1087-4e1d-b711-a06979649e18">

(I don't know if this is true, just throwing in here)

Reply:
`src/*.cpp` would match all cpp files in the `src/` directory and subdirectories. (This is what we want right now) We don't need to add `**` for subdirectories separately. 
Ref: https://cmake.org/cmake/help/latest/command/file.html#glob-recurse

Reply:
Sweet, thanks for confirming